bind.go

```diff
@@ -304,29 +308,32 @@ func (b *Bind) All(out any) error {
 	outElem := outVal.Elem()
 
 	// Precedence: URL Params -> Body -> Query -> Headers -> Cookies
-	sources := []func(any) error{
-		b.URI,
-		b.Body,
-		b.Query,
-		b.Header,
-		b.Cookie,
-	}
+	sources := []func(any) error{b.URI}
 
-	tempStruct := reflect.New(outElem.Type()).Interface()
+	// Check if both Body and Content-Type are set
+	if len(b.ctx.Request().Body()) > 0 && len(b.ctx.RequestCtx().Request.Header.ContentType()) > 0 {
+		sources = append(sources, b.Body)
+	}
+	sources = append(sources, b.Query, b.Header, b.Cookie)
+	prevSkip := b.skipValidation
+	b.skipValidation = true
```

Comment:
Not sure
What happens with parallel processing in the same App? Shouldn't we pass this on via arguments?

Reply:
her is no problem, the bind is only per request

Theoretically, problems could arise with multiple routes connected in series or an internal redirect, but these are edge cases.